src/rpc/masternode.cpp

```diff
@@ -406,6 +407,11 @@ static RPCHelpMan masternode_payments()
             if (tx->IsCoinBase()) {
                 continue;
             }
+            if (tx->IsPlatformTransfer()) {
```

Comment:
we don't have IsPlatformTransfer in v21.1.x

Reply:
Let's backport this function too, it's a single-liner

Reply:
Agree, looks like #6131 can be backported with no merge conflicts. Compiles with no issues too.